CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-07-20
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250720`.
+
+### Announcements
+* Kubernetes version 1.33 is now Generally Available for AKS. Patch version [1.33.1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.33.md#v1331) is also now supported.
+* Azure CNI Overlay with AGIC is now Generally Available. Customers can now use AGIC with Azure CNI Overlay without registering the AppGatewayWithOverlayPreview feature flag.
+* The asm-1-23 revision for the Istio add-on has been deprecated. Kindly upgrade your service mesh to a supported version following the [AKS Istio upgrade guide](https://learn.microsoft.com/en-us/azure/aks/istio-upgrade).
+* AKS Kubernetes version 1.33 is now compatible with Long-Term Support (LTS).
+* [Static Block allocation mode](https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni-static-block-allocation) for Azure CNI Networking is now Generally Available.
+* Virtual Machines (VMs) agent pools are now enabled by default.
+* The [WASI Node Pool](https://learn.microsoft.com/azure/aks/wasi-node-pool) feature has been deprecated with removal of the WasmNodePoolPreview feature flag.
+
+### Release notes
+* Features
+  * Application routing add-on now supports configuration of SSL passthrough, custom logging format, and load balancer IP ranges. Review the [configuration of NGINX ingress controller documentation](https://learn.microsoft.com/azure/aks/app-routing-nginx-configuration?tabs=azurecli#configuration-of-the-nginx-ingress-controller) for more information.
```

Comment:
Looks like a behavior change and not a feature.

Reply:
Moved it back to features

Reply:
@sabbour  - is this introduced in preview or is it GA?

Reply:
GA. It is a feature of the operator.